.github/workflows/hotfix-pr-check.yml

```diff
@@ -0,0 +1,159 @@
+name: Hotfix-PR-Check
+
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
```

Comment:
We might need to also include a "synchronize" trigger so that the workflow run does not get invalidated when new commits are pushed, and this is included as a required check.